test/e2e/memcached_test.go

```diff
@@ -218,9 +218,9 @@ func TestMemcached(t *testing.T) {
 	}
 
 	t.Log("Pulling new dependencies with go mod")
-	cmdOut, err = exec.Command("go", "mod", "vendor").CombinedOutput()
+	cmdOut, err = exec.Command("go", "build", "./...").CombinedOutput()
 	if err != nil {
-		t.Fatalf("Command 'go mod vendor' failed: %v\nCommand Output:\n%v", err, string(cmdOut))
+		t.Fatalf("Pulling modules failed: %v\nCommand Output:\n%v", err, string(cmdOut))
```

Comment:
Not sure pulling modules is the only error we could be running into, we should just say `go build failed` instead.